cli/installer/windows/local-build.ps1

```diff
@@ -0,0 +1,12 @@
+param(
+    $Version = "0.0.1"
+)
+
+$currentLocation = Get-Location 
+
+try { 
+    Set-Location $PSScriptRoot
```

Comment:
```suggestion
    Push-Location $PSScriptRoot
```

Reply:
I've generally avoided `Push` and `Pop` locations. I don't think they're going to cause problems in this case but if future iterations of this script include other scripts (or user defined script blocks) then it's possible that nested calls to `Push-Location` would not run the corresponding `Pop-Location` as expected and that would mean that our `Pop-Location` puts the user back in an unexpected state.

Reply:
Fair enough, but I would argue that's a bug on the callee script that should be fixed as well.